examples/splinterdb_intro_example.c

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+/*
+ * SplinterDB "Hello World" example program. Demonstrate use of:
+ *  - Basic SplinterDB configuration and create() interface
+ *  - Insert, lookup and iterator interfaces
+ *  - Close and reopen a SplinterDB db (instance)
+ */
+#include <stdio.h>
+#include <string.h>
+
+#include "splinterdb/default_data_config.h"
+#include "splinterdb/splinterdb.h"
+
+#define DB_FILE_NAME    "splinterdb_intro_db"
+#define DB_FILE_SIZE_MB 1024 // Size of SplinterDB device; Fixed when created
+#define CACHE_SIZE_MB   64   // Size of cache; can be changed across boots
+
+/* Application declares the limit of key-sizes it intends to use */
+#define APP_MAX_KEY_SIZE ((int)100)
+
+/*
+ * -------------------------------------------------------------------------------
+ * We, intentionally, do not check or show-for error handling, as this is mostly
```

Comment:
I don't understand this sentence. What does "show-for" mean here?

Reply:
Lazy commentary. Rephrased as:

```
 24  * We, intentionally, do not check for errors or show error handling, as this is
 25  * mostly a sample program to illustrative how-to use the APIs.
 ```